core/domain/html_cleaner.py

```diff
@@ -282,7 +276,10 @@ def convert_to_text_angular(html_data):
             tag.name = 'p'
         # All other invalid tags are simply removed.
         elif tag.name not in allowed_tag_list:
-            tag.unwrap()
+            try:
+                tag.unwrap()
+            except Exception:
+                return "exception: " + html_data
```

Comment:
Returning won't do, will it? You should raise here, catch it in the MR job, and yield the error message + html data.

Reply:
We are returning the html data as it is and since this will become an error string because `exception: ` part will fail the validation for unwrapped content and this will be added to list of invalid strings in `err_dict['strings']`. 
Could you please correct me here if I am wrong.

Reply:
Yep, I know. But that's implicit behaviour and is conceptually muddy. Errors should be logged in a different key in err_dict.

Reply:
Yes, I got it, I have fixed this now, PTAL